src/ui/public/visualize/visualize.js

```diff
@@ -55,6 +55,10 @@ uiModules
         });
 
         $scope.vis = $scope.savedObj.vis;
+        $scope.vis.createInheritedSearchSource = () => {
+          const searchSource = new $scope.vis.API.SearchSource().inherits($scope.savedObj.searchSource);
```

Comment:
I'm not sure if every savedObj will have a searchSource. Maybe check and throw something meaningful first?